scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -45,6 +82,23 @@ class ReducerEstimator {
 
 }
 
+trait HistoryReducerEstimator extends ReducerEstimator with HistoryService {
```

Comment:
We could easily refactor this to use composition instead of inheritance.
Should `HistoryReducerEstimator` be a class that takes `HistoryService` as a parameter?